eclair-core/src/main/resources/reference.conf

```diff
@@ -123,11 +123,14 @@ eclair {
 
   revocation-timeout = 20 seconds // after sending a commit_sig, we will wait for at most that duration before disconnecting
 
-  auth-timeout = 10 seconds // will disconnect if connection authentication doesn't happen within that timeframe
-  init-timeout = 10 seconds // will disconnect if initialization doesn't happen within that timeframe
-  ping-interval = 30 seconds
-  ping-timeout = 10 seconds // will disconnect if peer takes longer than that to respond
-  ping-disconnect = true // disconnect if no answer to our pings
+  peer-connection {
+    auth-timeout = 10 seconds // will disconnect if connection authentication doesn't happen within that timeframe
+    init-timeout = 10 seconds // will disconnect if initialization doesn't happen within that timeframe
+    ping-interval = 30 seconds
+    ping-timeout = 10 seconds // will disconnect if peer takes longer than that to respond
+    ping-disconnect = true // disconnect if no answer to our pings
+  }
+
   auto-reconnect = true
   initial-random-reconnect-delay = 5 seconds // we add a random delay before the first reconnection attempt, capped by this value
   max-reconnect-interval = 1 hour // max interval between two reconnection attempts, after the exponential backoff period
```

Comment:
Shouldn't these three also be in the peer connection section?

Reply:
No, those are related to the reconnection task. The scope of peer-connection is the handling of an already established connection (incoming or outgoing).